docs/layouts/shortcodes/integrations.html

```diff
@@ -1,20 +1,20 @@
-
+{{ $context := . }}
```

Comment:
What does this line do?

Reply:
redundant. Passing  `.` to relref is enough to get the context